calico_containers/tests/st/test_ipv6.py

```diff
@@ -0,0 +1,53 @@
+from sh import ErrorReturnCode
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestIpv6(TestBase):
+    def test_ipv6(self):
+        """
+        Test mainline functionality with IPv6 addresses.
+        """
+        host = DockerHost('host')
+
+        host.execute("docker run --rm  -v `pwd`:/target jpetazzo/nsenter", _ok_code=[0, 1])
+
+        calicoctl = "/code/dist/calicoctl %s"
+        host.execute(calicoctl % "node --ip=127.0.0.1 --ip6=fd80:24e2:f998:72d6::1")
```

Comment:
`fd80:24e2:f998:72d6::1` is the same IP as you use for the container.

It'd be better to add an `ip6` field to DockerHost (like the `ip` field it currently has).  You can hard-code it for now, until you get multi-host working.

Also, use `host.ip` for the IPv4 address, rather than `127.0.0.1`